cert/lib/cert/runner.rb

```diff
@@ -187,8 +187,9 @@ def create_certificate
       return cert_path
     end
 
-    def store_certificate(certificate)
-      path = File.expand_path(File.join(Cert.config[:output_path], "#{certificate.id}.cer"))
+    def store_certificate(certificate, filename = nil)
+      cert_name = filename ? filename : "#{certificate.id}.cer"
```

Comment:
Ah we already auto-append, right?

Reply:
There is no auto-append for now

Reply:
Here we need to choose a base filename and then append the `cer` extension (if not already on there) so I recommend a code change like the following...

```rb
cert_name = filename or certificate.id
cert_name = "#{cert_name}.cer" unless File.extname(base_name) == "cer"
```

⚠️ I wrote this code without testing it so it is subject to breakage 😇 But this should get you close to where we need to be

Reply:
Of course ! Minor updates : 
```ruby
cert_name = filename ? filename : certificate.id
cert_name = "#{cert_name}.cer" unless File.extname(cert_name) == ".cer"

Reply:
FYI, I rebase commits for keeping a clean history